pkg/image/api/types.go

```diff
@@ -106,6 +128,32 @@ type TagEvent struct {
 	DockerImageReference string
 	// The image
 	Image string
+	// Generation is the spec tag generation that resulted in this tag being updated
+	Generation int64
```

Comment:
If the `TagReference` can have this as `nil`, why can't the `TagEvent` also have this as nil?